### DISCUSSION TITLE
How to install Chrome OS 105?

### DISCUSSION THREAD

**Discussion by enthusiastic2003** [sebanc/brunch#1653] [General]
It has some useful features like game overlay. But the repo does not have its recovery image. What can i do?

**Comment by ameyjoshi3068** [comment]
Just try downloading and installing [brunch unstable](https://github.com/sebanc/brunch-unstable) on testing Machine

**Reply by sebanc** [reply]
It is supposed to be quite stable. Could you be more specific about the issues you are facing and add your device model / CPU / GPU ?

**Reply by nr3k** [reply]
I used the 105 unstable: https://github.com/sebanc/brunch-unstable/releases
and it works great
jjust to be honest, from v103 to v105 i gradually updated the brunch instead of doing it directrly from v103 to v105

**Reply by tejasraman** [reply]
@ameyjoshi3068 105 unstable is pretty stable for me. I have never had problems with unstable releases
